### PR TITLE
FAT-5729 Create JSON upload

### PR DESCRIPTION
**Pull request by Activict**

### CODE REVIEW

File: StoreApp/Util/FileParserJSON.cs [hunk 1]
```diff
@@ -0,0 +1,86 @@
+﻿using Newtonsoft.Json;
+using StoreApp.BLL.DTO;
+using StoreApp.Enums;
+using StoreApp.Models.Store;
+using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.IO;
+using System.Web;
+using System.Xml;
```
Comment (AleksandroBoko) [inline StoreApp/Util/FileParserJSON.cs]: Do you need all of these namespaces?

File: StoreApp/Util/FileParserJSON.cs [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+namespace StoreApp.Util
+{
+    public class FileParserJSON : IFileParser
+    {
+        private RootNames Type { get; set; }
+        private string Json { get; set; }
+        private HttpPostedFileBase File { get; set; }
```
Comment (AleksandroBoko) [inline StoreApp/Util/FileParserJSON.cs]: I suppose if we have private properties probably it's the best way to convert them into private fields. If you want we can discuss it.
So we can convert **private RootNames Type { get; set; }** into **private RootNames type**

File: StoreApp/Util/FileParserJSON.cs (hunk 1 above)
Comment (Activict) [reply StoreApp/Util/FileParserJSON.cs]: fixed

File: StoreApp/Util/FileParserJSON.cs (hunk 2 above)
Comment (Activict) [reply StoreApp/Util/FileParserJSON.cs]: fixed

**Pull request merged**